recipes/bamsurgeon/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "Bamsurgeon" %}
+{% set version = "1.4.1" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/adamewing/{{ name }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 137981f16df21fe5b4f602c641372a20643a8bb92e6c6dddace499427417213f
+
+build:
+  run_exports:
+    - {{ pin_subpackage('bamsurgeon', max_pin="x") }}
+  noarch: generic
+  number: 0
+
+requirements:

```

Comment:
wouldn't it make sense to use the requirements from: https://github.com/adamewing/bamsurgeon/blob/1885dacdde61c0e18bb30fa6af1af6507e975e1c/scripts/check_dependencies.py#L86-L92 ?